donate/payments/sqs.py

```diff
@@ -1,11 +1,17 @@
 from functools import lru_cache
 import json
-
+import logging
+import time
```

Comment:
just to prevent confusion: `from time import time, sleep` and then a bit further down you  don't need the kind of weird `time.time` (yay python naming =)